wa.js

```diff
@@ -114,6 +114,9 @@ Hooks.on("renderJournalSheet", (app, html, data) => {
       }
     }
   }
+});
+
+Hooks.on("renderJournalPageSheet", (app, html, data) => {
 
   // Activate cross-link listeners
   html.find(".wa-link").click(event => {
```

Comment:
It does not need to be part of this merge, but we have a new V10 feature for declaring custom text enrichments which can be used instead of this custom logic. 

You can add a `TextEditorEnricher` to `CONFIG.TextEditor.enrichers` which provides a regex pattern and a replacer callback function.

Reply:
Hoho,
Good to know. I will try to se how it works